utils/create_dummy_models.py

```diff
@@ -1395,7 +1416,7 @@ def create_tiny_models(
         raise ValueError(f"This script should be run from the root of the clone of `transformers` {clone_path}")
 
     report_path = os.path.join(output_path, "reports")
-    os.makedirs(report_path)
+    os.makedirs(report_path, exist_ok=True)
```

Comment:
before: we had to remove the reports folder each time we ran the command.
now: it overwrites the contents of the reports folder when we run the command.